test/syncenginetestutils.h

```diff
@@ -85,6 +90,14 @@ class DiskFileModifier : public FileModifier
     void mkdir(const QString &relativePath) override {
         _rootDir.mkpath(relativePath);
     }
+    void move(const QString &relativePath, const QString &relativeDestinationDirectory) override {
+        auto orig = _rootDir.filePath(relativePath);
+        auto dest = _rootDir.filePath(relativeDestinationDirectory);
+        QVERIFY(QFileInfo::exists(orig));
+        QVERIFY(QFileInfo::exists(dest));
+        QVERIFY(QFileInfo(dest).isDir());
+        QVERIFY(_rootDir.rename(relativePath, relativeDestinationDirectory));
```

Comment:
Move "A/a1" --> "B/a1" should probably not be rename to "B", or am I missing something? Is it a lot of work to add a test that moves a file to make sure it works too?